overview.md

```diff
@@ -96,10 +96,12 @@ When you use this task, all of the required files must be on the MATLAB search p
 Run all tests in a MATLAB project and generate test artifacts. Specify the task in your pipeline YAML using the `RunMATLABTests` key.
 
 Argument                  | Description    
-------------------------- | --------------- 
-`testResultsJunit`        | (Optional) Path to write test results report in JUnit XML format.<br/>**Example:** `'test-results/results.xml'`
+------------------------- | ---------------
 `codeCoverageCobertura`   | (Optional) Path to write code coverage report in Cobertura XML format.<br/>**Example:** `'code-coverage/coverage.xml'`
-`sourceFolder`      | (Optional) Location of the folder containing source code, relative to the project root folder. The specified folder and its subfolders are added to the top of the MATLAB search path. To generate a code coverage report, MATLAB uses only the source code in the specified folder and its subfolders. You can specify multiple folders using a colon-separated or a semicolon-separated list.<br/>**Example:** `'source'`
+`selectByFolder`          | (Optional) Location of the folder used to select test suite elements, relative to the project root folder. To generate a test suite, MATLAB uses only the tests in the specified folder and its subfolders. You can specify multiple folders using a colon-separated or a semicolon-separated list.<br/>**Example:** `'test/unit'`
+`selectByTag`             | (Optional) Test tag used to select test suite elements. To generate a test suite, MATLAB uses only the test elements with the specified tag.<br/>**Example:** `'Unit'`
+`sourceFolder`            | (Optional) Location of the folder containing source code, relative to the project root folder. The specified folder and its subfolders are added to the top of the MATLAB search path. To generate a code coverage report, MATLAB uses only the source code in the specified folder and its subfolders. You can specify multiple folders using a colon-separated or a semicolon-separated list.<br/>**Example:** `'source'`
```

Comment:
How about: 
If you specify `sourceFolder` and then choose to generate a code coverage report, ...

Reply:
The text for `sourceFolder` has not changed. I just moved it so the args are shown in alphabetical order. Do we still want to change it?

Reply:
No, it's not urgent. I just proposed the change because it makes the text clearer. Please feel free to leave it.